nc-news-fe/src/components/ArticleContainer.jsx

```diff
@@ -22,7 +23,11 @@ const ArticleContainer = () => {
     return (
         <>
             {articles.map(article => {
-                return <ArticleCard key={article.article_id} article={article} />
+                return (
+                    <Link key={article.article_id} to={`/${article.article_id}/${encodeURIComponent(article.title)}`}>

```

Comment:
Consider this url for articles with longer titles. As an example, this is a bbc news article url ```https://www.bbc.co.uk/news/world-europe-67679636```